app/lib/brexit_checker/group.rb

```diff
@@ -15,7 +15,7 @@ class BrexitChecker::Group
                                 studying-uk
                                 common-travel-area]
 
-  attr_reader :key, :heading
```

Comment:
The priority of each group is set in the [spreadsheet](https://docs.google.com/spreadsheets/d/1wIeBTitJVfkWa7oKrGmusIo2r4TsvXVdlne_xG6YjYs/edit#gid=443740486) but I can't see anywhere in the app that the priority is pulled in and used to set the order of the groups. It looks like the order of the elements in the yaml file is all that was previously determining the group order.